packages/typespec-ts/src/modular/buildOperationUtils.ts

```diff
@@ -0,0 +1,774 @@
+import { toPascalCase } from "../utils/casingUtils.js";
+import { importSettings } from "../utils/importUtils.js";
+import {
+  getResponseMapping,
+  getRequestModelMapping,
+  serializeRequestValue
+} from "./helpers/operationHelpers.js";
+import { ModularCodeModel, Property, Type } from "./modularCodeModel.js";
+import {
+  FunctionDeclarationStructure,
+  SourceFile,
+  StructureKind
+} from "ts-morph";
+import { Imports as RuntimeImports } from "@azure-tools/rlc-common";
+import { UsageFlags } from "@typespec/compiler";
+
+/**
+ * This function creates a file under /api for each operation group.
```

Comment:
Please ensure the comment is correct for this large file.

Reply:
It would be good that you could list the supported case that we could tell the difference between the models in unions.